lib/components/artist/shows.js

```diff
@@ -0,0 +1,44 @@
+/* @flow */
+'use strict';
+
+import React from 'react-native';
+const { View, Text, StyleSheet, Image } = React;
+
+import Headline from '../text/headline';
+import SerifText from '../text/serif'
+
+export default class Shows extends React.Component {
+
+	render() {
+	  return (
+	  	<View style={styles.container}>
+		  	<SerifText style={styles.title}> Current & Upcoming Shows </SerifText>
+		  	<Text style={styles.sansSerifText}> {this.props.show.location} </Text>
+		  	<Text style={styles.sansSerifText}> {this.props.show.partner_name} </Text>
```

Comment:
I can only imagine that the value of `children` isn’t a String. You can either `console.log(typeof (children))` or try to use the Chrome debugger to set a breakpoint and inspect it that way (I can screen-hero show you that tomorrow).